tests/utils/utils_test.go

```diff
@@ -22,20 +22,14 @@ func TestClientGo(t *testing.T) {
 }
 
 func TestTemplating(t *testing.T) {
-	tmpFilePath := createSparkOperatorNamespace("testtesttest")
+	tmpFilePath := createSparkJob(SparkJob{
+		Name:         "test",
+		Namespace:    "foo",
+		Image:        "bar",
+		SparkVersion: "baz",
+		Template:     "spark-linear-regression-job.yaml",
+	})
 	defer os.Remove(tmpFilePath)
 
 	log.Infof("Created a temp file at %s", tmpFilePath)
```

Comment:
should there be an assert in this test? Not sure what it's actually testing.

Reply:
This whole file is not included in test runs currently, i.e. not executed with `make test`. I'm using it mostly for debugging various utility methods. Later, when there are more complex logic in the tests, I plan to either turn it into proper tests or remove completely.